### PR TITLE
LPD-25878 Reindexing ConfigurationModel is slow

### PR DESCRIPTION
**Pull request by joshuacords**
Follow up to https://github.com/liferay-search/liferay-portal/pull/1345

https://liferay.atlassian.net/browse/LPD-25878

**Author:** @BryanEngler 
**Reviewer:** @joshuacords

### CODE REVIEW

**Comment by joshuacords** [comment]
ci:test:sf

**Comment by joshuacords** [comment]
ci:test:search

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationModelIndexer.java [hunk 1]
```diff
@@ -371,16 +369,7 @@ protected void doReindex(String[] ids) throws Exception {
...
-
-		for (ConfigurationModel configurationModel :
-				configurationModels.values()) {
-
-			documents.add(getDocument(configurationModel));
-		}
-
-		_indexWriterHelper.updateDocuments(
-			CompanyConstants.SYSTEM, documents, false);
+		reindex(configurationModels.values());
```
Comment (BryanEngler) [inline modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationModelIndexer.java:372]: ya i thought about doing this, the early returns kind of made me hesitate, didnt necessarily want to change the behavior too much. this is probably the right way to do it, i just wasnt able to look deep enough. in general this framework needs some attention, so we have to draw a line somewhere so we dont get carried away with the changes

**Comment by BryanEngler** [comment]
added a test https://github.com/liferay-search/liferay-portal/pull/1347

**Pull request closed without merging**